lib/matplotlib/cbook.py

```diff
@@ -2544,6 +2544,15 @@ def index_of(y):
 
 def safe_first_element(obj):
     if isinstance(obj, collections.Iterator):
+        # needed to accept `array.flat` is input.
```

Comment:
is -> as
And while I make that comment, the commit message also needs the same fix.